LabExT/View/Movement/LoadStoredCalibrationWindow.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+LabExT  Copyright (C) 2022  ETH Zurich and Polariton Technologies AG
+This program is free software and comes with ABSOLUTELY NO WARRANTY; for details see LICENSE file.
+"""
+import json
+
+from tkinter import W, Frame, Label, OptionMenu, StringVar, Toplevel, Button, messagebox, RIGHT, TOP, X, BOTH, FLAT, Y, LEFT
+from typing import Type
+from os.path import exists
+
+from LabExT.Utils import run_with_wait_window
+from LabExT.Movement.MoverNew import MoverNew
+from LabExT.View.Controls.CustomFrame import CustomFrame
+from LabExT.View.Controls.DeviceTable import DeviceTable
+from LabExT.Wafer.Chip import Chip, Device
+
+
+class LoadStoredCalibrationWindow(Toplevel):
+    """
+    Window to load stored calibrations
+    """
+
+    ASSIGNMENT_MENU_PLACEHOLDER = "-- unused --"
+
+    def __init__(
+        self,
+        master,
+        mover: Type[MoverNew],
+        chip: Type[Chip]
+    ) -> None:
+        """
+        Parameters
+        ----------
+        master : Tk
+            Tk instance of the master toplevel
+        mover : Mover
+            Instance of the current mover.
+        mover : Chip
+            Instance of the current chip.
+
+        Raises
+        ------
+        RuntimeError
+            If calibrations could not be restored.
+        """
+        self.mover: Type[MoverNew] = mover
+        self.chip: Type[Chip] = chip
+
+        if not messagebox.askyesno(
+            "Restore calibration",
+                f"Found mover calibration for chip: {self.chip.name}. Do you want to restore it?"):
```

Comment:
How do you know that we found a calibration at this point? There is no prior check if one is available, no?

Reply:
There was a check before the window gets called. But you are right, here it is missing, I added one.